Add country geojson to worldmap

mark-prins opened the pull request:
Adds a configurable geojson layer to the customised world map plugin.
Offroad: Have also made the icon look much ugly, so that it's easier to distinguish the custom version.
Offroad 2: also bumped versions of all packages while I was here.

Sample: 

![Screen Shot 2020-04-29 at 4 53 51 PM](https://user-images.githubusercontent.com/9192912/80562118-d97ff780-8a3a-11ea-8576-eaab4ddb377f.png)

Note that the fill is set to none, but the colour can be configured in the panel settings.

CarlosNZ commented:
Hi @mark-prins , have you got the geoJSON file/info handy to test this out? How do you enter it in the input field?

mark-prins commented:
This is the geojson text - one that I created by tracing the country border, deliberately lumpy in order to keep the file size down. I just copied/pasted as text into the input field

[laos.geojson.txt](https://github.com/openmsupply/msupply-dashboard/files/4556170/laos.geojson.txt)

CarlosNZ commented:
Looks real nice -- well done.

Could I just make one small suggestion, if it's not too complicated: it'd be good if the colour could accept rgba values, so you could specifiy an opacity/alpha value as well. I think the outline would look a little better if it was partially transparent, so you could make it blend into the background a bit more as desired.

It looks like the input field handles an rgb string okay (e.g. `rgb(244, 77, 51)`) -- just wonder how much extra work it would be to correctly interpret an rgba string (e.g. `rgba(0, 172, 16, 0.50)`) and handle the opacity value correctly?

No worries if it's too much for now -- I can always make a new issue to add this later if that's easier.

CarlosNZ commented on custom-plugins/grafana-worldmap-panel-msupply/src/worldmap_ctrl.ts:
```diff
@@ -1,60 +1,58 @@
-import $ from "jquery";
-import "./css/leaflet.css";
-import WorldMap from "./worldmap";
+import * as _ from 'lodash';
```
I notice most of the changes here are just minor re-formatting/style changes. I assume your chosen linter is just doing its thing, but should we be using a synced Prettier file or something to maintain consistency here? What's the best way to handle this going forward?

mark-prins replied:
oops. yes. just the defaults in my vscode which are causing the problems. I probably should have disabled. will look at adding in configs.

mark-prins commented:
thanks @CarlosNZ - I've added `.prettierrc` which has perversely removed the hanging commas.

Also removed the fill:false so that this now looks like the Laos example
